Makefile

```diff
@@ -5,13 +5,13 @@ ifdef CI
 	V ?= 1
 endif
 
-TERRAFORM_VERSION = 1.0.2
-TFLINT_VERSION = 0.29.1
-PACKER_VERSION = 1.7.3
+TF_VERSION = 1.0
+TFLINT_VERSION = 0.30.0
```

Comment:
this can be a breaking change what is the changes?

Reply:
TF 1.0 support - it's not  a breaking change